app/models/member.rb

```diff
@@ -8,7 +8,8 @@ class Member < ActiveRecord::Base
   def self.find_from_request(akid: nil, id: nil)
     if akid.present?
       actionkit_user_id = AkidParser.parse(akid)[:actionkit_user_id]
-      member = find_by(actionkit_user_id: actionkit_user_id)
+      #member = find_by(actionkit_user_id: actionkit_user_id)
+      member = where(actionkit_user_id: actionkit_user_id).order('created_at ASC').first
```

Comment:
not at all. I just forgot to remove it. I'd rather get this merged now, and I'll remove the line after, if that's ok (otherwise will have to wait for it to build and run tests again).